WcaOnRails/db/migrate/20180104132335_update_preferred_formats_for_feet.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class UpdatePreferredFormatsForFeet < ActiveRecord::Migration[5.1]
+  def change
+    PreferredFormat.where(event_id: "333ft").delete_all
```

Comment:
paranoia: `delete_all` -> `delete_all!`?

Reply:
The method doesn't exist :p

Reply:
Oh, sorry! I was thinking of `destroy_all!`. Using `delete_all` here seems fine to me!